descriptions/css/properties/cursor.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "cursor": {
+        "__short_description": "The <strong><code>cursor</code></strong> <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> property sets the cursor to show when the mouse pointer is over an element."
```

Comment:
Being picky, but I think this is potentially ambiguous "sets the cursor to be show" might be read as "sets whether or not to show the cursor". Perhaps "selects which type of cursor to show" or something like that?

Reply:
Well, it does both! Revised to cover that a bit more clearly